docs/src/Box.doc.js

```diff
@@ -419,11 +419,54 @@ card(
+      By default, the Box component renders a \`div\` element, which doesn't provide much meaning to the user or assistive technology. Use the \`as\` prop to inform which HTML component should be rendered by the Box component to ensure a meaningful experience.
 
-    Using the \`role\` property can prevent an excess of generic \`<div>s\` and give the user better context. Learn more about [ARIA roles](https://www.w3.org/TR/wai-aria/#usage_intro).
+      When using a Box component as a custom element, it is your responsibility to address all the accessibility implications. Both the \`role\` and \`as\` properties semantically classify the Box; however, the \`as\` prop defines a more concise way to describe the HTML element by modifying the underlying DOM element directly, which helps support both accessibility and SEO. We recommend using the \`as\` prop when possible, making sure that the prop type is semantically associated with the Box content.
+
+      Review the [available options for the as prop](#Props). For some of the options, like \`nav\`, you will also need to specify a \`title\` to ensure unique landmarks on the page.
+    `}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Flex direction="column" flex="grow">
+  <Flex alignItems="top" >
+    <Box as="nav" title="as prop example nav" column={6} color="maroon" width="100%" padding={2} borderStyle="sm">
+      <Text color="white" weight="bold">Top Nav Menu: as="nav"</Text>
+    </Box>
+    <Box column={6} display="inlineBlock" borderStyle="sm">
+      <Box color="midnight" width="100%" padding={2}>
+        <Text color="white" weight="bold">HTML output:</Text>
+        <Text color="white" weight="bold">{"<nav>Menu</nav>"}</Text>
+      </Box>
+    </Box>
+  </Flex>
+  <Flex alignItems="top">
+    <Box as="article" column={6} color="pine" width="100%" padding={2} borderStyle="sm">
+      <Heading color="white" size="md">Article 1</Heading>
+      <Text color="white" weight="bold">Article: as="article"</Text>
+    </Box>
+    <Box column={6} display="inlineBlock" borderStyle="sm">
+      <Box color="midnight" width="100%" padding={2}>
+        <Text color="white" weight="bold">HTML output:</Text>
+        <Text color="white" weight="bold">{"<article>"}</Text>
+        <Text color="white" weight="bold">   {"<h2>Article 1</h2>"}</Text>
+        <Text color="white" weight="bold">{"</article>"}</Text>
+      </Box>
+    </Box>
+  </Flex>
+</Flex>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Using 'role' property"
+      description={`
+    Setting the  \`role\` property on Box classifies the Box as the semantically appropriate HTML element through the use of an ARIA role while leaving the underlying element as a \`div\`. For example, setting \`role="banner"\` will designate that Box to be the equivalent of a \`<header>\` within the page, allowing assistive technology to classify the Box appropriately.
```

Comment:
the equivalent of a \`<header>\` within the page _element hierarchy_? maybe?

Reply:
assistive technology or technologies?

Reply:
I believe it can be assistive technology, since we are talking in general and not giving specific examples.
The idea is to define assistive technology is any device, software, or equipment that helps people with disabilities learn, communicate, or function better.